### DISCUSSION TITLE
for request limiting endpoint of an API

### DISCUSSION THREAD

**Discussion by rishav2404** [nfriedly/express-rate-limit#467] [Q&A]
can this package be used to limit total no. of request on a endpoint of API ? If yes then how and if not then what to look after. Please suggest!!

**Comment by gamemaker1** [comment]
Hi @rishav2404,

Yes, it can be used to limit the total number of requests to a certain API endpoint of an Express app in a given duration of time. See the [installation](https://express-rate-limit.mintlify.app/quickstart/installation) and [usage](https://express-rate-limit.mintlify.app/quickstart/usage) pages of the documentation for a detailed guide on how to do the same.
